.travis.yml

```diff
@@ -1,9 +1,7 @@
 language: python
 python:
-  - "3.3"
-  - "3.4"
-  - "3.5"
-  - "3.6"
+  - "2.7"
+  - "3.6.6"
```

Comment:
May be worth running checks in future for 3.6.8 and 3.7+, but not essential at this stage.